starter-api/src/main/java/io/micronaut/starter/api/create/AbstractCreateController.java

```diff
@@ -80,6 +81,9 @@ protected AbstractCreateController(
             @Nullable Language lang,
             @Nullable JdkVersion javaVersion,
             String userAgent) {
+        if (name.equalsIgnoreCase(MICRONAUT)) {
+            throw new HttpStatusException(HttpStatus.BAD_REQUEST, "project name cannot be " + name);
+        }
```

Comment:
Wouldn't it be better to check the project name directly in `NameUtils.parse(name)`?

https://github.com/micronaut-projects/micronaut-starter/pull/403/files#diff-f5c89fa2c105c0ad496c0386652c7389L84-L88